src/coreclr/src/tools/Common/Internal/Runtime/ReadyToRunInstructionSet.cs

```diff
@@ -0,0 +1,131 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// DO NOT EDIT THIS FILE! IT IS AUTOGENERATED
+using System;
+using System.Runtime.InteropServices;
+using Internal.TypeSystem;
+
+namespace Internal.ReadyToRunConstants
+{
+    public enum ReadyToRunInstructionSet
+    {
+        Sse=1,
+        Sse2=2,
+        Sse3=3,
+        Ssse3=4,
+        Sse41=5,
+        Sse42=6,
+        Avx=7,
+        Avx2=8,
+        Aes=9,
+        Bmi1=10,
+        Bmi2=11,
+        Fma=12,
+        Lzcnt=13,
+        Pclmulqdq=14,
+        Popcnt=15,
+        ArmBase=16,
+        AdvSimd=17,
+        Crc32=18,
+        Sha1=19,
+        Sha256=20,
+        Atomics=21,
+
+    }
+
+    public static class ReadyToRunInstructionSetHelper
+    {
+        ReadyToRunInstructionSet? R2RInstructionSetFromJitInstructionSet(TargetArchitecture architecture, Internal.JitInterface.InstructionSet instructionSet)
+        {
+            switch (architecture)
+            {
+
+                case TargetArchitecture:
+                    {
+                        switch (instructionSet)
+                        {
+                            case InstructionSet.ARM64_ArmBase: return ReadyToRunInstructionSet.ArmBase;
+                            case InstructionSet.ARM64_ArmBase_Arm64: return ReadyToRunInstructionSet.ArmBase;
+                            case InstructionSet.ARM64_AdvSimd: return ReadyToRunInstructionSet.AdvSimd;
+                            case InstructionSet.ARM64_AdvSimd_Arm64: return ReadyToRunInstructionSet.AdvSimd;
+                            case InstructionSet.ARM64_Aes: return ReadyToRunInstructionSet.Aes;
+                            case InstructionSet.ARM64_Crc32: return ReadyToRunInstructionSet.Crc32;
+                            case InstructionSet.ARM64_Crc32_Arm64: return ReadyToRunInstructionSet.Crc32;
+                            case InstructionSet.ARM64_Sha1: return ReadyToRunInstructionSet.Sha1;
+                            case InstructionSet.ARM64_Sha256: return ReadyToRunInstructionSet.Sha256;
+                            case InstructionSet.ARM64_Atomics: return ReadyToRunInstructionSet.Atomics;
+                            case InstructionSet.ARM64_Vector64: return ReadyToRunInstructionSet.;
```

Comment:
Missing a `Base` entry?

Reply:
Heh, you caught me, this file isn't actually attached to the build yet. Once this is merged, I'll rebase my existing ISA changes to use this stuff, and … well I'll have to fix this problem. I'd rather not invest too much on the R2R instruction set stuff at this time in this PR as I don't have the code in the branch that will use any of it.

Reply:
I've fixed this, even though this file still isn't attached to the build.